index.html

```diff
@@ -86,41 +78,169 @@ <h2 class="second-header">I am software developer</h2>
+          </div>
+        </div>
+        <div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </div><div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </div><div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </div><div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </div><div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <div class="card-description">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
           </div>
         </div>
       </div>
     </section>
     <section class="about-me">
       <div class="about-me-description">
```

Comment:
- [ ] Please  the button should follow the button styling: Active, Hover, and Pressed

![image](https://user-images.githubusercontent.com/74744787/159043025-dcc67890-71e9-426c-9c8c-2b6dcb374589.png)